doc/reference/reference_lua/box_events/system_events.rst

```diff
@@ -0,0 +1,126 @@
+
+*   ``instance_uuid``: the UUID of the instance never changes after the first
+    :doc:`box.cfg </reference/reference_lua/box_cfg>`.
+    The value is unknown before the ``box.cfg`` call.
+
+*   ``replicaset_uuid``: the value is unknown until the instance joins a replicaset or boots a new one.
+    The events start working before that -- right with the start of the listen.
+
+..  code-block:: lua
+
+    -- box.id value
+    {
+    MP_STR “id”: MP_UINT; box.info.id,
+    MP_STR “instance_uuid”: MP_UUID; box.info.uuid,
+    MP_STR “replicaset_uuid”: MP_UUID box.info.cluster.uuid,
+    }
+
+box.status
+~~~~~~~~~~
+
+Contains generic blob about the instance status.
+
+*   ``is_ro``: :ref:`indicates the read-only mode <box_introspection-box_info>` or the ``orphan`` status.
+*   ``is_ro_cfg``: indicates the :ref:`read_only <cfg_basic-read_only>` mode for the instance.
+*   ``status``: shows the status of an instance.
+
+..  code-block:: lua
+
+    {
+    MP_STR “is_ro”: MP_BOOL box.info.ro,
+    MP_STR “is_ro_cfg”: MP_BOOL box.cfg.read_only,
+    MP_STR “status”: MP_STR box.info.status,
+    }
+
+box.election
+~~~~~~~~~~~~
+
+Contains fields of the :doc:`box.info.election </reference/reference_lua/box_info/election>`
+that are necessary to find out the most recent writable leader.
+
+*   ``term``: shows the current election term.
+*   ``role``: indicates the election state of the node -- ``leader``, ``follower``, or ``candidate``.
+*   ``is_ro``: :ref:`indicates the read-only mode <box_introspection-box_info>` or the ``orphan`` status.
+*   ``leader``: shows the leader node ID in the current term.
+
+..  code-block:: lua
+
+    {
+    MP_STR “term”: MP_UINT box.info.election.term,
+    MP_STR “role”: MP_STR box.info.election.state,
+    MP_STR “is_ro”: MP_BOOL box.info.ro,
+    MP_STR “leader”: MP_UINT box.info.election.leader,
+    }
+
+box.schema
+~~~~~~~~~~
+
+Contains schema-related data.
+
+*   ``version``: shows the schema version.
+
+..  code-block:: lua
+
+    {
+    MP_STR “version”: MP_UINT schema_version,
+    }
+
+Usage example
+-------------
+
+..  code-block:: lua
+
+    conn = net.box.connect(URI)
+    -- Subscribe to updates of key 'box.id'
+    w = conn:watch('box.id', function(key, value)
+        assert(key == 'box.id')
+        -- do something with value
+    end)
+    -- or to updates of key 'box.status'
+    w = conn:watch('box.status', function(key, value)
+        assert(key == 'box.status')
+        -- do something with value
+    end)
+    -- or to updates of key 'box.election'
+    w = conn:watch('box.election', function(key, value)
+        assert(key == 'box.election')
+        -- do something with value
+    end)
+    -- or to updates of key 'box.schema'
+    w = conn:watch('box.schema', function(key, value)
+        assert(key == 'box.schema')
+        -- do something with value
+    end)
+    -- Unregister the watcher when it's no longer needed.
+    w:unregister()
```

Comment:
Wrong example.

1. a user should be able to copy-paste. So he will copy unregister, too and will think: 'Why it doesn't work?'
So example of unregister should be moved to the other example.
2. Use local variables (not global) in example: `local w`, `local conn`

Reply:
Yes, it's a good idea to use `local` in examples. It doesn't work in the REPL console that we like to use, but the interactive console is not the most popular method of communication with Tarantool. Our clients rely on Lua programs, and using the `local` scope is a best practice for safety.